src/ansys/additive/core/single_bead.py

```diff
@@ -126,7 +106,7 @@ def material(self, value):
     def bead_length(self) -> float:
         """Length (m) of bead to simulate.
 
-        Valid values are from :obj:`MIN_BEAD_LENGTH` to :obj:`MAX_BEAD_LENGTH` m.
+        Valid values are from :obj:`MIN_BEAD_LENGTH` to :obj:`MAX_BEAD_LENGTH` meters.
```

Comment:
```suggestion
        Valid values are from the :obj:`MIN_BEAD_LENGTH` value to the :obj:`MAX_BEAD_LENGTH` value.
```
Same question here about meters.